src/map/utils/petutils.cpp

```diff
@@ -1462,6 +1462,15 @@ namespace petutils
             {
                 ((CItemWeapon*)PPet->m_Weapons[SLOT_MAIN])->setDelay((uint16)(floor(1000.0 * (280.0f / 60.0f))));
             }
+  
+            if (PetID == PETID_CARBUNCLE)
+            {
+                if (PPet->GetMLevel() >= 25)
+                {
+                    PPet->setModifier(Mod::REGEN, 1);
+                }
+            }
```

Comment:
This can be simplified to:
```cpp
if (PetID == PETID_CARBUNCLE && PPet->GetMLevel() >= 25)
{
    PPet->setModifier(Mod::REGEN, 1);
}
```

Reply:
There is no mention of the regen getting stronger at higher levels.
I left it expanded in case there is any other carbuncle mods that also need to be added.

Reply:
even simpler is to see if he has MDB and if he does, set mainjob to whm because 2 traits being whm traits is evidence enough for me.

Reply:
> There is no mention of the regen getting stronger at higher levels. I left it expanded in case there is any other carbuncle mods that also need to be added.

the code snippet isn't scaling anything like that, its just combining your checks